Sources_v3/StreamChat/APIClient/Endpoints/CurrentUserEndpoints.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+extension Endpoint {
+    static func updateCurrentUser<ExtraData: UserExtraData>(
+        id: UserId,
+        payload: CurrentUserUpdateRequestBody<ExtraData>
+    ) -> Endpoint<CurrentUserUpdateResponse<ExtraData>> {
```

Comment:
Since we provide `UserId` I don't think this needs to be named `updateCurrentUser`. Technically, you can update any user you want if you have rights to do that. So maybe just `updateUser`?

Reply:
Then, should this be moved to the UserEndpoints? Instead of having a `CurrentUserEndpoints`?